### PR TITLE
Create eeee.php

### PR DESCRIPTION
**Pull request by anologicon**

### CODE REVIEW

**Comment by m31317015** [comment]
Maybe e.php instead?

File: eeee.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+$e = true;
```
Comment (alexandrecruz) [inline eeee.php]:
```suggestion
$e = 'e';
```

File: eeee.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+while ($e) {
```
Comment (alexandrecruz) [inline eeee.php]:
```suggestion
while ($e == $e) {
```

Comment (alexandrecruz) [reply eeee.php]: eeee eeee, e eee? #455

**Review by alexandrecruz — CHANGES_REQUESTED** [review]
Eeeee eee eeeee. ee eee eeeeee eee??

eeee >> e, e?

File: eeee.php (hunk 2 above)
Comment (anologicon) [reply eeee.php]: `$e == $e` ???

Comment (alexandrecruz) [reply eeee.php]: `$e == $e` === true

Theeee moree eeeeee beeeetteeer :D

**Comment by anologicon** [comment]
oh, have another e.php, i'm close this PR ok ?

**Review by alexandrecruz — APPROVED** [review]

**Pull request closed without merging**